media/whip_server.go

```diff
@@ -310,6 +304,20 @@ func handleRTP(ctx context.Context, segmenter *RTPSegmenter, timeDecoder *rtptim
 				clog.Info(ctx, "empty nalus", "len", len(d), "payload", len(p.Payload), "seq", p.SequenceNumber)
 				continue
 			}
+
+			for _, nalu := range nalus {
```

Comment:
not from this PR but would it be worth putting a comment for the line `nalus, err := splitH264NALUs(d)` to explain what that's doing? I don't know what NALUs are sorry.

Reply:
They are the [H.264 unit](https://en.wikipedia.org/wiki/Network_Abstraction_Layer) that RTP streams deal with, from the first paragraph of [RFC 6184](https://datatracker.ietf.org/doc/html/rfc6184)

> The RTP payload format allows for packetization of one or more
> Network Abstraction Layer Units (NALUs), produced by an H.264 video
> encoder, in each RTP payload.

Reply:
Dropped a link to the RFC in https://github.com/livepeer/go-livepeer/pull/3540/commits/065ccf15ae80095d00818dda5778d2e70c3cfc43 , most of the code here will not make sense without understanding that anyway